Doc/library/asyncio-eventloop.rst

```diff
@@ -218,17 +224,18 @@ Scheduling callbacks
 
    An optional keyword-only *context* argument allows specifying a
    custom :class:`contextvars.Context` for the *callback* to run in.
-   The current context is used when no *context* is provided.
+   Callbacks use the current context when no *context* is provided.
 
-   An instance of :class:`asyncio.Handle` is returned, which can be
+   Returns an instance of :class:`asyncio.Handle`, which can be
```

Comment:
```suggestion
   Return an instance of :class:`asyncio.Handle`, which can be
```

Reply:
Particularly if we frame it this way, this sentence/para would seem to fit best immediately after the first/summary line. This also follows the loose convention used other places in this module (e.g. in the following `call_later` method) and in other modules.